lib/ansible/modules/windows/win_dsc.py

```diff
@@ -30,24 +30,21 @@
 module: win_dsc
 version_added: "2.4"
 short_description: Invokes a PowerShell DSC configuration
-description: |
-     Invokes a PowerShell DSC Configuration. Requires PowerShell version 5 (February release or newer).
-     Most of the parameters for this module are dynamic and will vary depending on the DSC Resource.
-     In order to find the required parameters for a given DSC resource, you can use the following on-liner:
-     'Get-DscResource <dsc_resource> | select -ExpandProperty properties'
-     Also note that credentials are handled as follows: If the resource accepts a credential type property called "cred",
-     the ansible parameters would be cred_username and cred_password.
-     These will be used to inject a credential object on the fly for the DSC resource.
+description:
+     - Invokes a PowerShell DSC Configuration. Requires PowerShell version 5 (February release or newer).
+     - Most of the parameters for this module are dynamic and will vary depending on the DSC Resource.
+     - In order to find the required parameters for a given DSC resource, you can use the following on-liner: 'Get-DscResource <dsc_resource> | select -ExpandProperty properties'
```

Comment:
I think this is failing because of the `: `
`- In order to find the required parameters for a given DSC resource, you can use the following  C(Get-DscResource <dsc_resource> | select -ExpandProperty properties)`

That:s
* Remove the `: `
* Use `C(...)` for inline code

See http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#formatting-options for background